confignode/src/main/java/org/apache/iotdb/confignode/conf/ConfigNodeConfCheck.java

```diff
@@ -70,10 +70,10 @@ public void checkConfig() throws RepeatConfigurationException, IOException {
       }
     }
 
-    FileInputStream inputStream = new FileInputStream(specialPropertiesFile);
-    specialProperties.load(inputStream);
-    checkSpecialProperties();
-    inputStream.close();
+    try (FileInputStream inputStream = new FileInputStream(specialPropertiesFile)) {
```

Comment:
I think maybe the `try` statement is useless. Because if there occurs an IOException, it will be throwed outside, and the ConfigNode won't be started

Reply:
closed where it is used. Because we can't control where this method will be called later, if the caller gets an exception and does not kill the process, a link leak will occur.